.github/workflows/reusable-build.yml

```diff
@@ -21,31 +26,6 @@ jobs:
       contents: read
       packages: write
       id-token: write
-    strategy:
-      fail-fast: false
-      matrix:
-        fedora_version:
-          - ${{ inputs.fedora_version }}
-        image_name:
-          - silverblue
-          - kinoite
-          - sericea
-          - onyx
-          - base
-          - lazurite
-          - vauxite
-        include:
-          - image_name: base
-            source_image: ${{ inputs.fedora_version == '41' && 'base-atomic' || '' }}
-          - image_name: sericea
-            source_image: ${{ inputs.fedora_version == '41' && 'sway-atomic' || '' }}
-          - image_name: onyx
-            source_image: ${{ inputs.fedora_version == '41' && 'budgie-atomic' || '' }}
-          - image_name: lazurite
-            source_image: ${{ inputs.fedora_version == '41' && 'lxqt-atomic' || '' }}
-          - image_name: vauxite
-            source_image: ${{ inputs.fedora_version == '41' && 'xfce-atomic' || '' }}
```

Comment:
All matrix logic is in the parent workflows so you don't need any conditionals here